### ISSUE TITLE
Sprint: Success & Next Steps

### ISSUE THREAD

**Issue by xmatthewx** [mozilla/network#22]
We held a work week to advance: Content, Marketing (e.g. Value proposition), Communications (Internal/External), Design UX/UI, Engagement planning. 

I'll use this issue to ensure proper documentation and that next steps are assigned. 

A huge thanks to @HPaulJohnson @KevZawacki @taisdesouzalessa @ldecoursy for kickin' butt and making all of this possible. cc @chrislarry33 

Shoutout to @gvn @sabrinang @openmatt @prefontaine for their awesome work in parallel.

## Successes

Project wide
 - Drafted external dependencies list #23

Communications
 - Internal comms plan advanced
 - Comms to network plan initiated
 - Sent update/announcement to staff
 - Identified open questions on MoFo/MLN social media profiles 

Content
 - Drafted editorial calendar
 - Identified and drafted offerings for launch CTAs. Identified owners.
 - Identified launch Medium and earned media items
 - Finalized v1 copy, including hero options to test
 - Set requirements for staff/board/member profiles
 - Advanced work to gather content for staff, board, member profiles

Marketing & Engagement
 - Refined statement on audience, goals, value prop
 - Mapped user journey
 - Submitted name options and rationale
 - Drafted post-launch testing plan. Phases for value proposition, hero & CTAs, join #48 
 - Oriented staff who will contribute to post-signup engagement
 - Mapped flow of Offerings to DB

Design
 - Drafted wireframes for join page template
 - Added and crafted a News page for earned media
 - Locked down people profile card UX design
 - UI draft complete
 - Unblocked design relative to new brand

## Next steps 

 - coming soon

**Comment by xmatthewx** [comment]
Closing! Next steps are either in motion or no longer relevant.
